apps/vs-code-designer/src/app/commands/createNewCodeProject/createCodeProjectSteps/createLogicApp/CodelessFunctiontWorkflow.ts

```diff
@@ -0,0 +1,246 @@
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import {
+  hostFileName,
+  azureWebJobsStorageKey,
+  localSettingsFileName,
+  workflowFileName,
+  workflowType,
+  localEmulatorConnectionString,
+  extensionBundleId,
+  defaultVersionRange,
+} from '../../../../../constants';
+import { localize } from '../../../../../localize';
+import { setLocalAppSetting } from '../../../../utils/appSettings/localSettings';
+import {
+  addFolderToBuildPath,
+  addNugetPackagesToBuildFile,
+  getDotnetBuildFile,
+  suppressJavaScriptBuildWarnings,
+  updateFunctionsSDKVersion,
+  writeBuildFileToDisk,
+} from '../../../../utils/codeless/updateBuildFile';
+import { getFramework, validateDotnetInstalled } from '../../../../utils/dotnet/executeDotnetTemplateCommand';
+import { writeFormattedJson } from '../../../../utils/fs';
+import { parseJson } from '../../../../utils/parseJson';
+import { WorkflowCreateStepBase } from '../../../createCodeless/createCodelessSteps/WorkflowCreateStepBase';
+import type { IActionContext } from '@microsoft/vscode-azext-utils';
+import { DialogResponses, nonNullProp, parseError } from '@microsoft/vscode-azext-utils';
+import { WorkflowProjectType, MismatchBehavior } from '@microsoft/vscode-extension';
+import type { IFunctionWizardContext, IWorkflowTemplate, IHostJsonV2, StandardApp } from '@microsoft/vscode-extension';
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import type { MessageItem } from 'vscode';
+
+// This class creates a new workflow for a codeless Azure Function project
+export class CodelessFunctiontWorkflow extends WorkflowCreateStepBase<IFunctionWizardContext> {
+  // Private constructor to prevent direct instantiation of this class
+  private constructor() {
+    super();
+  }
+
+  // Static method that creates a new instance of the CodelessFunctionProjectWorkflowCreateStep class and returns it
+  public static async createStep(context: IActionContext): Promise<CodelessFunctiontWorkflow> {
+    // Ensure that the .NET Core SDK is installed on the user's machine
+    await validateDotnetInstalled(context);
+    return new CodelessFunctiontWorkflow();
+  }
+
+  // Async method that creates a new workflow for the codeless Azure Function project
+  public async executeCore(context: IFunctionWizardContext): Promise<string> {
+    // Get the function template and function name from the IFunctionWizardContext object
+    const template: IWorkflowTemplate = nonNullProp(context, 'functionTemplate');
+    const functionPath: string = path.join(context.projectPath, nonNullProp(context, 'functionName'));
+    const methodName = context.methodName;
+
+    // Create empty stateful and stateless definition objects
+    const emptyStatefulDefinition: StandardApp = {
```

Comment:
I can't remember, can you make sure if it is desire to have an already created schema or an empty schema its fine